src/diffusers/__init__.py

```diff
@@ -239,6 +239,9 @@
             "LatentConsistencyModelImg2ImgPipeline",
             "LatentConsistencyModelPipeline",
             "LDMTextToImagePipeline",
+            "LEditsPPPipelineIF",
```

Comment:
Is it intentional that the first "E" is capitalized? Wouldn't `LeditsppPipelineIF` be better?

Reply:
Yes it is, since the Method itself is also capitalized that wey "LEdits++", similar to lets say "IF" capitalization.